app/utils/constants.ts

```diff
@@ -147,6 +147,27 @@ const PROVIDER_LIST: ProviderInfo[] = [
     getApiKeyLink: 'https://lmstudio.ai/',
     labelForGetApiKey: 'Get LMStudio',
     icon: "i-ph:cloud-arrow-down",
+  },
+  {
+    name: 'NovitaAI',
+    staticModels: [
+      { name: 'qwen/qwen-2.5-72b-instruct', label: 'qwen-2.5-72b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16000 },
+      { name: 'meta-llama/llama-3.1-405b-instruct', label: 'llama-3.1-405b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16384 }
+    ],
+    getApiKeyLink: 'https://novita.ai/settings/api-keys'
+  },
+  {
+    name: 'TogetherAI',
+    staticModels: [
+      { name: 'Qwen/Qwen2.5-72B-Instruct-Turbo', label: 'Qwen2.5-72B-Instruct-Turbo', provider: 'TogetherAI', maxTokenAllowed: 1000 },
+      { name: 'meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo', label: 'Llama 3.1 8B Instruct Turbo', provider: 'TogetherAI', maxTokenAllowed: 1000 },
```

Comment:
Any url to get API keys?
Seems like its here https://api.together.xyz/settings/api-keys

Reply:
the url to get the api key is 
https://[api.together.ai/settings/api-keys](https://api.together.ai/settings/api-keys)

you need to register and 1$ credit free to test llm models.